tailwind.config.js

```diff
@@ -109,6 +109,18 @@ module.exports = {
         '.special-hyphens-auto': {
           hyphens: 'auto',
         },
+        '.special-text-h3': {
+          fontSize: '1.3125rem',
+          lineHeight: '1.22',
+        },
+        '.special-text-h4': {
+          fontSize: '1.1875rem',
+          lineHeight: '1.22',
+        },
+        '.special-text-2x': {
+          fontSize: '2rem',
+          lineHeight: '1.22',
+        },
```

Comment:
see https://tailwindcss.com/docs/font-size#customizing

Reply:
much to learn I have

Reply:
Should be solved!